src/components/RadioButton/__stories__/radioButton.stories.mdx

```diff
@@ -61,62 +75,94 @@ Radio buttons have 3 states: regular, selected, and disabled.
+            <Link href="/?path=/docs/inputs-checkbox--overview" withoutSpacing>
+              checkboxes
+            </Link>{" "}
+            instead.
+          </>
+        )
       }
     },
     {
       positive: {
-        component:
-        <DialogContentContainer className="monday-style-radio-buttons_wrapper">
-            <RadioButton text="Item 1" name="radio-buttons-group-2" checked />
+        component: (
+          <DialogContentContainer className="monday-style-radio-buttons_wrapper">
+            <RadioButton text="Item 1" name="radio-buttons-group-2" defaultChecked />
             <RadioButton text="Item 2" name="radio-buttons-group-2" />
             <RadioButton text="Item 3" name="radio-buttons-group-2" />
-          </DialogContentContainer>,
-        description:"Mark the first item as selected, and make sure it’s the most common or popular action."
+          </DialogContentContainer>
+        ),
+        description: "Mark the first item as selected, and make sure it’s the most common or popular action."
       },
       negative: {
-        component:
-        <DialogContentContainer className="monday-style-radio-buttons_wrapper">
+        component: (
+          <DialogContentContainer className="monday-style-radio-buttons_wrapper">
             <RadioButton text="Item 1" name="radio-buttons-group-3" />
             <RadioButton text="Item 2" name="radio-buttons-group-3" />
             <RadioButton text="Item 3" name="radio-buttons-group-3" />
-        </DialogContentContainer>,
-        description:<>Don’t leave all radio buttons unselcted. If you can’t offer a smart default, consider using the <Link href="/?path=/docs/inputs-dropdown--overview" withoutSpacing>dropdown component.</Link></>
+          </DialogContentContainer>
+        ),
+        description: (
+          <>
+            Don’t leave all radio buttons unselcted. If you can’t offer a smart default, consider using the{" "}
+            <Link href="/?path=/docs/inputs-dropdown--overview" withoutSpacing>
+              dropdown component.
+            </Link>
+          </>
+        )
       }
     }
   ]}
 />
 
 ## Use cases and examples
+
 ### Radio button in items list
+
 The user needs to select only one option.
+
 <Canvas>
-  <Story name="Radio button in items list">
+  <Story name="Radio button in items list" play={clickRadioButton}>
     <div className="monday-style-radio-buttons_wrapper-column">
       <div>Inbox view options</div>
-      <RadioButton text="Inbox updates" name="radio-buttons-group-4" checked />
+      <RadioButton text="Inbox updates" name="radio-buttons-group-4" defaultChecked />
       <RadioButton text="I was mentioned" name="radio-buttons-group-4" />
       <RadioButton text="All updates" name="radio-buttons-group-4" />
     </div>
   </Story>
 </Canvas>
 
+Controled externally.
+
+<Canvas>
+  <Story name="Controlled Radio buttons" play={controlRadioButton}>
+    {controlledRadioButton.bind({})}
```

Comment:
Developer will not be able to see the code this way. The best thing to do will be to put this code as part of the mdx file (example in combobox stories ). if it's not possible can you please use ControlledRadioButton as the template? (for developers be able to see the code when click on "show code")